### PR TITLE
Mock KlerosLiquid

### PR DESCRIPTION
**Pull request by bbenligiray**
Closes https://github.com/api3dao/claims-manager/issues/25

I will be working on the tests next, which should uncover any issues with this. So you don't need to do a very deep review.

### CODE REVIEW

File: contracts/mock/MockKlerosLiquid.sol [hunk 1]
```diff
@@ -0,0 +1,385 @@
...
+pragma solidity ^0.8.0;
+
+/* solhint-disable */
+// A standalone contract that mocks below
+// https://github.com/kleros/kleros/blob/master/contracts/kleros/KlerosLiquid.sol
+// Names starting with __ do not exist in KlerosLiquid and added for convenience here.
+// Events are omitted for brevity.
+// Set the metadata to start with subcourt 1 with 3 minimum number of jurors.
+// All staking/voting functionality is stripped, use __setCurrentRuling() instead.
+// Use __setSubcourtTimesPerPeriod() if you need to speed up the flow.
```
Comment (mcoetzee) [inline contracts/mock/MockKlerosLiquid.sol:11]: I see the name of the implementing function is `changeSubcourtTimesPerPeriod`, I guess it should be `__setSubcourtTimesPerPeriod`?

File: contracts/mock/MockKlerosLiquid.sol [hunk 2]
```diff
@@ -0,0 +1,385 @@
...
+        uint256[4] calldata _timesPerPeriod
+    ) external {
+        __courts[_subcourtID].timesPerPeriod = _timesPerPeriod;
+    }
+
+    function __setCurrentRuling(uint256 disputeId, uint256 ruling) external {
+        currentRuling[disputeId] = ruling;
+    }
+
+    function passPeriod(uint256 _disputeID) external {
```
Comment (mcoetzee) [inline contracts/mock/MockKlerosLiquid.sol:127]: Ah ok so if I understand correctly, my [`give-dispute-ruling`](https://github.com/api3dao/api3-dao-dashboard/blob/main/hardhat/hardhat.config.ts#L209) Hardhat task would update to:

1. set zero period lengths for the first couple of periods (the ones preceding the `appeal` period)
2. call `__setCurrentRuling()` to give the ruling
3. call `passPeriod()` a couple of times to get into the `appeal` period

Does that sound about right?

**Review by mcoetzee — APPROVED** [review]
🥂

File: contracts/mock/MockKlerosLiquid.sol (hunk 1 above)
Comment (bbenligiray) [reply contracts/mock/MockKlerosLiquid.sol:11]: I did initially use `__setSubcourtTimesPerPeriod()`, but then noticed that there is already a `changeSubcourtTimesPerPeriod()` in KlerosLiquid so I overrode it. `__setSubcourtTimesPerPeriod()` may make more sense though.

File: contracts/mock/MockKlerosLiquid.sol (hunk 2 above)
Comment (bbenligiray) [reply contracts/mock/MockKlerosLiquid.sol:127]: Yeah. I figured calling `passPeriod()` once in a while won't be too much of a hassle.

**Pull request merged**